envs/differential-stats.yaml

```diff
@@ -0,0 +1,14 @@
+channels:
```

Comment:
Maybe, now that this is well established, we should aim to pin most versions to the current ones in the working environments. But we can maybe do that on a separate PR.

Reply:
Or at least avoid that they go beyond the next major version.